pyproject.toml

```diff
@@ -95,7 +112,10 @@ convention = "google"
 
 [tool.setuptools]
 zip-safe = false
+packages = ["fake_useragent"]
+
+[tool.setuptools.package-dir]
+fake_useragent = "src/fake_useragent"
 
 [tool.setuptools.package-data]
-"fake_useragent.data" = ["*.jsonl"]
-"fake_useragent" = ["py.typed"]
+"fake_useragent" = ["data/*.jsonl", "*py.typed"]
```

Comment:
Are you sure this works? So the setuptools scans the src directory first, and then finds the data directory: https://setuptools.pypa.io/en/latest/userguide/datafiles.html#subdirectory-for-data-files

Reply:
This works but probably not needed; I'm reverting some changes, thanks!